l1-contracts/contracts/bridgehub/Bridgehub.sol

```diff
@@ -329,6 +333,17 @@ contract Bridgehub is IBridgehub, ReentrancyGuard, Ownable2StepUpgradeable, Paus
                              Getters
     //////////////////////////////////////////////////////////////*/
 
+    /// @notice Asset Handlers' token address function, which takes assetId as input and return the token address
+    function tokenAddress(bytes32 _baseTokenAssetId) public view returns (address) {
```

Comment:
@Raid5594 this is a new function no?

Reply:
Also the baseToken function's interface changed, it should be chainId, I'll fix it